src/daemons/rhsmcertd.c

```diff
@@ -690,7 +659,6 @@ deprecated_arg_init_config (Config * config, int argc, char *argv[])
     }
 
     config->cert_interval_seconds = atoi (argv[1]) * 60;
-    config->heal_interval_seconds = atoi (argv[2]) * 60;
```

Comment:
since this is removed you need to tweak the `argc` check above:
```c
    if (argc != 3) {
```
... which must be 2 now

Reply:
Addressed by commit [a8aedb6](https://github.com/candlepin/subscription-manager/pull/3455/commits/a8aedb6a6f8dacce552cdee4b664f6d72d62b924)